.che-didact-guide.md

```diff
@@ -0,0 +1,37 @@
+
+
```

Comment:
FYI these 2 extra empty lines do not seem to do anything

Reply:
thanks ... I was desperately looking for end of line :)